<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Music</class>
 <widget class="QWidget" name="Music">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>721</width>
     <height>431</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>350</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget{color: White;
								background-color:rgba(0,0,0,0.7);
								border:2px groove gray;
								border-radius:10px;
								padding:2px 4px;}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_musicback">
   <property name="geometry">
    <rect>
     <x>1410</x>
     <y>740</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{color: Black;
								background-color:rgba(255,255,255,0.75);
								border:2px groove gray;
								border-radius:10px;
								padding:2px 4px;}</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>450</y>
     <width>631</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">QLabel{color:Black;}</string>
          </property>
          <property name="text">
           <string>播放方式</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="mode">
          <property name="minimumSize">
           <size>
            <width>72</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QComboBox{color: White;
								background-color:rgba(0,0,0,0.75);
								border:2px groove gray;
								border-radius:10px;
								padding:2px 4px;}</string>
          </property>
          <item>
           <property name="text">
            <string>顺序播放</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>随机播放</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>单曲循环</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_playing">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{color:black;}</string>
          </property>
          <property name="text">
           <string>正在播放：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="music_name">
          <property name="styleSheet">
           <string notr="true">QLabel{color:black;}</string>
          </property>
          <property name="text">
           <string>歌曲名</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QSlider" name="horizontalSlider">
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="pageStep">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="positionLabel">
          <property name="text">
           <string>00:00</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="start">
          <property name="styleSheet">
           <string notr="true">QPushButton{color: White;
								background-color:rgba(0,0,0,0.75);
								border:2px groove gray;
								border-radius:10px;
								padding:2px 4px;}</string>
          </property>
          <property name="text">
           <string>开始</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pause">
          <property name="styleSheet">
           <string notr="true">QPushButton{color: White;
								background-color:rgba(0,0,0,0.75);
								border:2px groove gray;
								border-radius:10px;
								padding:2px 4px;}</string>
          </property>
          <property name="text">
           <string>暂停</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stop">
          <property name="styleSheet">
           <string notr="true">QPushButton{color: White;
								background-color:rgba(0,0,0,0.75);
								border:2px groove gray;
								border-radius:10px;
								padding:2px 4px;}</string>
          </property>
          <property name="text">
           <string>停止</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="add">
          <property name="styleSheet">
           <string notr="true">QPushButton{color: White;
								background-color:rgba(0,0,0,0.75);
								border:2px groove gray;
								border-radius:10px;
								padding:2px 4px;}</string>
          </property>
          <property name="text">
           <string>选择歌曲路径</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>721</width>
     <height>741</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
								background-color:rgba(255,255,255,0.85);
								border:2px groove gray;
								border-radius:10px;
								padding:2px 4px;}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QSlider" name="verticalSlider">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>570</y>
      <width>21</width>
      <height>100</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>0</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="vol">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>680</y>
     <width>100</width>
     <height>36</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{color:black;}</string>
   </property>
   <property name="text">
    <string>音量：50</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>vol</zorder>
  <zorder>frame</zorder>
  <zorder>tableWidget</zorder>
  <zorder>btn_musicback</zorder>
  <zorder>layoutWidget</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
